src/components/Dialog/Dialog.stories.tsx

```diff
@@ -21,12 +22,23 @@ const DialogStory = (args: IDialogProps) => {
   return (
     <>
       <Button onPress={() => setOpen(true)}>Open Dialog</Button>
-      <Dialog
-        {...args}
-        onCancel={closeDialog}
-        onConfirm={closeDialog}
-        open={open}
-      />
+      <Dialog {...args} onCancel={closeDialog} open={open}>
+        <Dialog.Title>Discard draft?</Dialog.Title>
+        <Text variant="body">Discard draft?</Text>
```

Comment:
On the web, we have a DialogContent component for modal content

Reply:
You're right, we don't actually need it. Then we need to remove it from the web library